src/types/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './api';
+export * from './errors';
+export * from './registration';
```

Comment:
Does external devs need types from this module?

Reply:
Sure. See usage in playground and test app.